pkg/core/util.go

```diff
@@ -94,6 +94,18 @@ func deployNativeContracts(magic netmode.Magic) *transaction.Transaction {
 	return tx
 }
 
+func commiteeFromConfig(cfg config.ProtocolConfiguration) ([]*keys.PublicKey, error) {
+	validators := make([]*keys.PublicKey, cfg.ValidatorsCount)
```

Comment:
If this method is supposed to return the whole commitee, we shouldn't restrict it with validators count. Now this method performs the same operation as `validatorsFromConfig`.

Reply:
Fixed.